pkg/monitors/vsphere/metadata.yaml

```diff
@@ -0,0 +1,1093 @@
+        group: cpu
+      vsphere.cpu_costop_ms:
+        description: Time the virtual machine is ready to run, but is unable to run due to co-scheduling constraints.
+        type: counter
+        default: false
+        group: cpu
+      vsphere.cpu_demand_mhz:
+        description: The amount of CPU resources a virtual machine would use if there were no CPU contention or CPU
+          limit.
+        type: counter
+        default: false
+        group: cpu
+      vsphere.cpu_demand_entitlement_ratio_percent:
+        description: CPU resource entitlement to CPU demand ratio.
+        type: gauge
+        default: false
+        group: cpu
+      vsphere.cpu_entitlement_mhz:
+        description: CPU resources devoted by the ESXi scheduler.
+        type: gauge
+        default: false
+        group: cpu
+      vsphere.cpu_idle_ms:
+        description: Total time that the CPU spent in an idle state.
+        type: counter
+        default: false
+        group: cpu
+      vsphere_cpu_latency_percent:
+        description: Percent of time the virtual machine is unable to run because it is contending for access to the
+          physical CPU(s).
+        type: gauge
+        default: false
+        group: cpu
+      vsphere.cpu_maxlimited_ms:
+        description: Time the virtual machine is ready to run, but is not running because it has reached its maximum CPU
+          limit setting.
+        type: counter
+        default: false
+        group: cpu
+      vsphere.cpu_overlap_ms:
+        description: Time the virtual machine was interrupted to perform system services on behalf of itself or other
+          virtual machines.
+        type: counter
+        default: false
+        group: cpu
+      vsphere.cpu_readiness_percent:
+        description: Percentage of time that the virtual machine was ready, but could not get scheduled to run on the
+          physical CPU.
+        type: gauge
+        default: false
+        group: cpu
+      vsphere.cpu_ready_ms:
+        description: Time that the virtual machine was ready, but could not get scheduled to run on the physical CPU
+          during last measurement interval. CPU ready time is dependent on the number of virtual machines on the host
+          and their CPU loads.
+        type: counter
+        default: false
+        group: cpu
+      vsphere.cpu_reservedCapacity_mhz:
+        description: Total CPU capacity reserved by virtual machines.
+        type: gauge
+        default: false
+        group: cpu
+      vsphere.cpu_run_ms:
+        description: Time the virtual machine is scheduled to run
+        type: counter
+        default: false
+        group: cpu
+      vsphere.cpu_swapwait_ms:
+        description: CPU time spent waiting for swap-in.
+        type: gauge
+        default: false
+        group: cpu
+      vsphere.cpu_system_ms:
+        description: Amount of time spent on system processes on each virtual CPU in the virtual machine.
+        type: counter
```

Comment:
We use `cumulative` for cumulative counters in the metadata.

Reply:
Thx. Replaced all.

Reply:
Make sure the value being emitted is actually a cumulative and not just a delta incremental value.

Reply:
Reverted these to `counter` based on feedback from Ben. Basically, everything is a `datapoint.Gauge` except for `PerfStatsTypeDelta` which is a `datapoints.Count`.